sdk/batch/azure-batch/azure/batch/models/_models.py

```diff
@@ -63,7 +63,7 @@ class AccountListSupportedImagesOptions(Model):
 
     :param filter: An OData $filter clause. For more information on
      constructing this filter, see
-     https://docs.microsoft.com/rest/api/batchservice/odata-filters-in-batch#list-support-images.
+     https://docs.microsoft.com/en-us/rest/api/batchservice/odata-filters-in-batch#list-support-images.
```

Comment:
sorry, just confused why you're adding back 'en-us' in this commit, lmk if I'm missing ontext

Reply:
https://github.com/Azure/azure-sdk-for-python/pull/12697#issuecomment-663632732 This code is autogenerated so it won't make a difference if i change here, it will be overridden